script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_5B.txt

```diff
@@ -60,42 +60,42 @@ Arcueid's plunge back then was extraordinary.
 So I believe it's definitely possible that Vlov can be pinned down.
 However―――
 <Page17>
-"―――But, still―――"
+"―――But, still―――"//to be rechecked tl suggestion 1: replace 'still' with 'even so', 2: replace 'still' with 'even so' (note: the word used needs to be the same on both lines if a change is made.), 3: suggested new version: Is Arcueid's view of him still correct?
```

Comment:
leave this line as is, drop the still on the next line and make it "...Is Vlov in good condition?" it doesn't need to be a word-for-word reflection here.